cmp/pinpad/PinPadModel.js

```diff
@@ -1,3 +1,9 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2020 Extremely Heavy Industries Inc.
+ */
 import FastClick from '@onsenui/fastclick';
```

Comment:
bummer to bring the onsen code in to desktop apps obviously.  assume its actually needed on the table?
Will that import other onsen code?

Reply:
Let me take a look and see if I can get it imported on into the mobile version